submissions/cat-folloe-mouse-real/catFollowMouse.js

```diff
@@ -0,0 +1,281 @@
+(function oneko() {
```

Comment:
This is clearly a copy of [oneko.js](https://github.com/adryd325/oneko.js/blob/main/oneko.js). Copying other people's work is not acceptable.